lm/util.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Tuple
+
+import i6_core.rasr as rasr
+
+
+def _has_image(c: rasr.RasrConfig, pc: rasr.RasrConfig):
```

Comment:
I think I would either like to spell config and post_config out or have a small docstring here.

Reply:
I agree with @Atticus1806